source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -302,10 +302,10 @@ Fields:
 * roundTripTime: the duration of the ismaster call. Default null.
 * lastWriteDate: a 64-bit BSON datetime or null.
   The "lastWriteDate" from the server's most recent ismaster response.
-* opTime: an ObjectId or null.
-  The last opTime reported by the server; an ObjectId or null.
+* opTime: an opTime or null.
+  An opaque value representing the position in the oplog of the most recent write. Default null.
```

Comment:
Can we use the same phrasing as [the max-staleness spec](https://github.com/mongodb/specifications/blob/master/source/max-staleness/max-staleness.rst#lastwrite) to keep the definitions in sync?

> opTime: an opaque value representing the most recent replicated write. ...

Reply:
Yeah, it would be best to keep them in line with each other. It looks like that one might need to be updated too, though, since I don't think `opTime` gives any guarantees on whether the write was replicated or not. For that, you'd need [`majorityOpTime`](https://docs.mongodb.com/manual/reference/command/isMaster/#isMaster.lastWrite.majorityOpTime) (I think).

Reply:
I've updated them both to a common definition, let me know if it seems accurate.